src/components/DashboardNav.js

```diff
@@ -1,35 +1,79 @@
-import React, { Component } from "react";
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
 import { NavLink, withRouter } from "react-router-dom";
+import NotificationTip from "../login/components/NotificationTip/NotificationTip";
 import i18n from "../login/translation/InjectIntl_HOC_factory";
 
-class DashboardNav extends Component {
-  render() {
-    return (
-      <nav id="dashboard-nav">
-        <ul>
-          <NavLink exact activeClassName="active" to={`/profile/`}>
-            <li>{this.props.translate("dashboard_nav.profile")}</li>
-          </NavLink>
-          <NavLink
-            exact
-            activeClassName="active"
-            to={`/profile/verify-identity/`}
-          >
-            <li>{this.props.translate("dashboard_nav.identity")}</li>
-          </NavLink>
-          <NavLink activeClassName="active" to={`/profile/settings/`}>
-            <li>{this.props.translate("dashboard_nav.settings")}</li>
-          </NavLink>
-          <NavLink
-            activeClassName="active"
-            to={`/profile/settings/advanced-settings`}
-          >
-            <li>{this.props.translate("dashboard_nav.advanced-settings")}</li>
-          </NavLink>
-        </ul>
-      </nav>
-    );
+function DashboardNav(props) {
+  const [active, setActive] = useState(false);
+  const nins = useSelector(state => state.nins.nins);
+  const phones = useSelector(state => state.phones.phones);
+  const verifiedNin = nins.filter(nin => nin.verified);
+  const verifiedSweNumber = phones.filter(phone => phone.verified && phone.number.includes("+46"));
+  const unverifiedNumber = phones.filter(phone => !phone.verified);
+  // depending on languages show different styles
+  const selectedLanguage = props.intl.locale;
+  
+  let tipsAtIdentity = "";
+  let tipsAtSettings = "";
+  // if user does not added id number, rendering text on IDENTITY tab, only posible to verify with a freja eId
+  if(!nins.length){
```

Comment:
The first/outmost if clause for this functionality ought to be "if (!verifiedNin.length)", making the intent clearer.